README.adoc

```diff
@@ -11,7 +11,7 @@ image:{img-qg-status}[Quality Gate Status,link={uri-qg-status}]
 *This project is analysed on https://sonarcloud.io/dashboard?id=sonarsource-cfamily-examples_linux-autotools-travis-sc[SonarCloud]!*
 
 :note-caption: :information_source:
-NOTE: This simple example ignores https://sonarcloud.io/documentation/analysis/languages/cfamily/#analysis-cache[analysis cache] and https://sonarcloud.io/documentation/analysis/languages/cfamily/#multithreaded-code-scan[multithreaded execution] both of which can boost analysis significantly.
+NOTE: This simple example ignores https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-cache[analysis cache] and https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#parallel-code-scan[multithreaded execution] both of which can boost analysis significantly.
 
 It is very easy to analyze a C, C++ and Objective-C project with SonarCloud https://docs.travis-ci.com/user/sonarcloud/[on Travis CI]:
```

Comment:
In another cfamily Travis example, this link points to https://travis-ci.org/ which makes more sense in my opinion, as there is already a link to https://docs.travis-ci.com/user/sonarcloud/ 2 lines below

Reply:
Do you refer to this example: https://github.com/sonarsource-cfamily-examples/linux-cmake-travis-sc ?
In that example, the "on Travis CI" also refers to the doc page rather than the Travis landing page.

It is debatable, IMO, it make sense to link to the doc page, since user will likely know what Travis is when reading this readme, and will look for more specific information, i.e., about SC integration. If they don't know, they will be able to get to the Travis landing page in their second click on the navigation pane on that doc page.

The link is repeated down below because the "Documentation" section aims at gathering all the relevant links in one place. There it also has more opportunity to describe the link.

Finally, in all other SC examples I pass through, the link in this place (i.e., analyze C, C++ and Objective-C project with SonarCloud _on Your CI_) always points to the integration documentation rather than the landing page of the corresponding CI.

Reply:
Yes, I referred to this example. I thought the first link was to the main page, but I was mistaken; sorry about that.
I agree about the documentation section; I mentioned the link under "Travis Add-on" just after. 
This README has three links that point to the same Travis SC documentation... I suppose it's better this way than the other